packages/flutter/test/gestures/drag_test.dart

```diff
@@ -24,7 +22,7 @@ void main() {
       didStartPan = true;
     };
 
-    Offset updatedScrollDelta;
+    Offset? updatedScrollDelta;
```

Comment:
In this case it can't. It is only updated when when the pan updates. There is a check on line 46 to see if it is null after just a down event, signifying that the update wasn't yet called.